proto/ethermint/evm/v1/evm.proto

```diff
@@ -30,6 +30,9 @@ message Params {
   // active_static_precompiles defines the slice of hex addresses of the precompiled
   // contracts that are active
   repeated string active_static_precompiles = 10;
+  // denom_decimals defines the amount of decimals of the evm denom
+  uint32 denom_decimals = 11
```

Comment:
I believe we should merge the evm_denom and denom_decimal into a new type since they are related.

Reply:
this will break backwards compatibility. We'll need to add some fallback for all places were we're using the evm denom from the params for historical queries not to break